src/opera/parser/tosca/v_1_3/service_template.py

```diff
@@ -56,25 +56,41 @@ def normalize(cls, yaml_node):
         }, yaml_node.loc)
 
     @classmethod
-    def parse(cls, yaml_node, base_path, csar_path):
+    def parse(cls, yaml_node, base_path, template_path, parsed_templates):
         service = super().parse(yaml_node)
-        service.visit("prefix_path", csar_path)
-        service.merge_imports(base_path)
-        return service
+        if template_path is None:
```

Comment:
Under what conditions can this be `None`? This is puzzling me a bit.

Reply:
Well yes this can be annoying but previously this parameter was named `csar_path` and I think that parameter could be None. For example look here: https://github.com/xlab-si/xopera-opera/blob/master/tests/opera/parser/tosca/v_1_3/test_service_template.py#L51, where the two parameters passed to the parse method are None.

Now I had to add additional parameter for `parsed_templates` to the tests and I used an empty set for that (set()). But still other params like `base_path` and `template_name` are None when testing service template: https://github.com/xlab-si/xopera-opera/blob/import-once/tests/opera/parser/tosca/v_1_3/test_service_template.py#L51

Reply:
Well, you know what this means? That we need to fix the tests because I was lazy the first time around ;) The themplate path should aways contain something sensible.

Reply:
Ok, so now I used `tmp_path` for both `base_path` and `template_name` in tests to prevent them from being None.